src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.Manifest/WorkloadManifest.targets

```diff
@@ -42,5 +53,8 @@
         <Import Project="Sdk.props" Sdk="Microsoft.NET.Runtime.Emscripten.Node" />
         <Import Project="Sdk.props" Sdk="Microsoft.NET.Runtime.Emscripten.Sdk" />
         <Import Project="Sdk.props" Sdk="Microsoft.NETCore.App.Runtime.AOT.Cross.browser-wasm" />
+        <Import Project="Sdk.props" Sdk="Microsoft.NETCore.App.Runtime.browser-wasm.Props.Sdk" />
     </ImportGroup>
+
+    <Import Project="Sdk.targets" Sdk="Microsoft.NET.Runtime.MonoTargets.Sdk" />
```

Comment:
You need a condition here or these targets will be imported *everywhere*, this file is essentially part of the .NET SDK.

Reply:
What would the condition be?  We want this used everywhere, don't we?

Reply:
Move it into the mportGroup for each of the platforms we support.  Otherwise, it will be used in places where we aren't using mono.